core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -140,6 +148,11 @@ private RpcUtils() {} // prevent instantiation
       throw new InternalException(e).toGrpcStatusException();
     } finally {
       MetricsSystem.counter(getQualifiedInProgressMetricName(methodName)).dec();
+      Instant endTime = Instant.now();
+      long timeElapsed = Duration.between(startTime, endTime).toMillis();
+      if (timeElapsed > RPC_TIME_WARNING_THESHOLD) {
```

Comment:
If you want to calculate the cost for each rpc call, `us` would be more accurate.

From this PR, we can create a new metrics, lets say it `rpcProcessingTime`

Reference https://github.com/apache/hadoop/blob/trunk/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/metrics/RpcMetrics.java